extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/main/java/io/quarkus/resteasy/reactive/links/deployment/LinksProcessor.java

```diff
@@ -188,7 +188,9 @@ private RuntimeValue<GetterAccessorsContainer> implementPathParameterValueGetter
                                 entityType, getterMetadata.getFieldName(), getterMetadata.getGetterAccessorName());
                     }
                 }
+
             }
+            getterAccessorsContainerRecorder.validateContainer(getterAccessorsContainer);
```

Comment:
I don't understand why we're delaying the validation check until startup via a recorder. We can validate this in the build phase, no?

Reply:
I though this was the build phase. If not, then I honestly don't know where that is 😅 
Either way the validation I added is specifically for a field called `id`.  But given that just a field annotated with `@Id` will also work, I'm not seeing how that can be validated :/